contracts/CrossChain/Strategy2Crv.sol

```diff
@@ -0,0 +1,211 @@
+                        _sell(address(rewardsToken), rewardsTarget, _rewardsBalance);
+                    }
+                }
+            }
+        }
+
+        // If there is dai or usdc add liquidity to get more want
+        uint256 _dai = IERC20(dai).balanceOf(address(this));
+        uint256 _usdc = IERC20(usdc).balanceOf(address(this));
+        if (_dai > 0 || _usdc > 0) {
+            ICurveFi(curve).add_liquidity([_dai, _usdc], 0);
+        }
+
+        // debtOustanding will only be > 0 in the event of revoking or if we need to rebalance from a withdrawal or lowering the debtRatio
+        if (_debtOutstanding > 0) {
+            if (_stakedBal > 0) {
+                // don't bother withdrawing if we don't have staked funds
+                gauge.withdraw(Math.min(_stakedBal, _debtOutstanding));
+            }
+
+            uint256 _withdrawnBal = balanceOfWant();
+            _debtPayment = Math.min(_debtOutstanding, _withdrawnBal);
+        }
+
+        // serious loss should never happen, but if it does (for instance, if Curve is hacked), let's record it accurately
+        uint256 assets = estimatedTotalAssets();
+        uint256 debt = vault.strategies(address(this)).totalDebt;
+
+        // if assets are greater than debt, things are working great!
+        if (assets > debt) {
+            _profit = assets.sub(debt);
+            uint256 _wantBal = balanceOfWant();
+            if (_profit.add(_debtPayment) > _wantBal) {
+                // this should only be hit following donations to strategy
+                liquidateAllPositions();
+            }
+        }
+        // if assets are less than debt, we are in trouble
+        else {
+            _loss = debt.sub(assets);
+        }
+
+        // we're done harvesting, so reset our trigger if we used it
+        forceHarvestTriggerOnce = false;
+    }
+
+    // Sells our harvested CRV into the selected output.
+    function _sell(address token, address target, uint256 _amount) internal {
+        uint256 _balance = IERC20(token).balanceOf(address(this));
+        uint256 _tokenAllowance = _tradeFactoryAllowance(token);
+        if (_balance > _tokenAllowance) {
+            _createTrade(token, target, _balance - _tokenAllowance, tradeSlippage, block.timestamp + 604800);
+        }
+    }
+
+    /* ========== KEEP3RS ========== */
+
+    // convert our keeper's eth cost into want
+    function ethToWant(uint256 _ethAmount)
+        public
+        view
+        override
+        returns (uint256)
+    {
+        // Not needed
+        return _ethAmount;
+    }
+
+    /* ========== SETTERS ========== */
+
+    // These functions are useful for setting parameters of the strategy that may need to be adjusted.
+
+    // Use to add or update rewards
+    function updateRewards(address _rewardsToken) external onlyGovernance {
+        rewardsToken = IERC20(_rewardsToken);
+        // update with our new token, use dai as default
+        rewardsTarget = address(dai);
```

Comment:
rewardsTarget = address(dai) < I think this can be removed

Reply:
done